### PR TITLE
Blog - Deploy Applications to AWS ECR with a GitHub Actions CI/CD Pipeline

### PR DESCRIPTION
**Pull request by msali123**

### CODE REVIEW

**Comment by msali123** [comment]
[Preview Link](https://642d91f2106c0251973b0c71--ephemeral-narwhal-3ac3b2.netlify.app/blog/aws-ecr-github-actions/)
@balapriyac

File: blog/_posts/2023-05-25-aws-ecr-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+
+You require a packaged application that is easily portable throughout the pipeline. Docker provides a consistent and isolated environment for building, testing, and deploying applications. It allows you to create Docker images that package an application and its dependencies into a single portable artifact. This makes it easier to trigger changes across different stages of the pipeline.
+
+To manage a [Docker](/blog/rails-with-docker) image, you need infrastructures allowing you to store and retrieve images while maintaining application scalability. AWS provides [ECR](https://aws.amazon.com/ecr/) as a fully managed container registry service to deploy your application using Docker.
+
+Let's dive in and create a GitHub Actions workflow that will automate deployments to [AWS ECR](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html).
+
+### Prerequisites
+
+Before proceeding with this guide, you require the following:
```
Comment (balapriyac) [inline blog/_posts/2023-05-25-aws-ecr-github-actions.md:37]:
```suggestion
To follow along with this guide, you require the following:
```

File: blog/_posts/2023-05-25-aws-ecr-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+Go ahead and refresh your ECR repository, and your application image will be deployed as follows:
+
+<div class="wide">
+![Image guide]({{site.images}}{{page.slug}}/jfi5YV3.png)\
+</div>
+
+## Conclusion
+
+Deploying applications to AWS ECR with a GitHub Actions CI/CD creates a reliable pipeline that automates Docker builds and [deployment](/blog/deployment-strategies) cycles. This guide helped you learn how to deploy an application to AWS ECR using a GitHub Actions CI/CD pipeline. I hope you found the GitHub Actions workflow useful while leveraging automation to AWS resources. To further improve your [CI/CD pipelines](https://earthly.dev/blog/ci-vs-cd/), leverage other AWS services, such as [CodePipeline](https://aws.amazon.com/codedeploy/) and [CodeDeploy](https://aws.amazon.com/codepipeline/), and streamline your pipeline infrastructure.
+
```
Comment (balapriyac) [inline blog/_posts/2023-05-25-aws-ecr-github-actions.md:433]:
```suggestion

Deploying applications to AWS ECR with a GitHub Actions CI/CD creates a reliable pipeline that automates Docker builds and [deployment](/blog/deployment-strategies) cycles. This guide helped you learn how to deploy an application to AWS ECR using a GitHub Actions CI/CD pipeline. I hope you found the GitHub Actions workflow useful while leveraging automation to AWS resources.

To further improve your [CI/CD pipelines](https://earthly.dev/blog/ci-vs-cd/), leverage other AWS services, such as [CodePipeline](https://aws.amazon.com/codedeploy/) and [CodeDeploy](https://aws.amazon.com/codepipeline/), and streamline your pipeline infrastructure.

```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks, Sameer. Just a couple small changes.

**Comment by balapriyac** [comment]
@jalletto this is ready to merge.

**Review by jalletto — APPROVED** [review]

**Pull request merged**